Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -365,8 +363,8 @@ which has the default route on the host. To change the device, set its name in
 the ``global.nodePort.device`` helm option.
 
 In addition, thanks to the :ref:`host-services` feature, the NodePort service can
-be accessed by default from a host or a pod within a cluster via it's public,
-cilium_host device or loopback address, e.g. ``127.0.0.1:NODE_PORT``.
+be accessed by default from a host or a pod within a cluster via it's public or
+loopback address, e.g. ``127.0.0.1:NODE_PORT``.
```

Comment:
Within the cluster accessing via cilium_host device will actually still work since it will use the wildcard entry and the cilium_host device IP is in the ipcache as HOST_ID. So should say: ` via it's public, any local or loopback [...]`

Reply:
(This is only for access from outside world.)

Reply:
I thought about adding `any local`. However, not every local IP addr has HOST_ID, e.g. an IP addr of `docker0` is considered as WORLD_ID.

Reply:
Discussed offline: `docker0` was a special case. Updated the docs accordingly.